ApplicationSettingSecond.page

```diff
@@ -0,0 +1,23 @@
+<apex:page StandardController="Student__c" Extensions="StudentRegistrationController" language="{!languageCode}">
+<apex:pageMessages id="abc"></apex:pageMessages>
+    <apex:form >
+      <apex:selectList label="Language" value="{!selectedLang}" size="1" multiselect="false">
+           <apex:selectOptions value="{!langList}"/>
+           <apex:actionSupport event="onchange" action="{!selectChange}" rerender="abc, student"/>
```

Comment:
Translation is not working for all components of page is only works with field label.